Makefile

```diff
@@ -9,17 +9,25 @@ PREFIX ?= ${ROOT}/usr
 INSTALL_DIR ?= ${PREFIX}
 MODULE_DIR := ${ROOT}/modules
 
-DEB_COMPONENT := clearwater-snmp-handlers
-DEB_MAJOR_VERSION := 1.0${DEB_VERSION_QUALIFIER}
+PKG_COMPONENT := clearwater-snmp-handlers
+PKG_MAJOR_VERSION ?= 1.0
+
+# Define DEB_NAMES and RPM_NAMES separately as we don't have RPMs for all the
+# components yet.
 DEB_NAMES := clearwater-snmp-handler-cdiv clearwater-snmp-alarm-agent clearwater-snmp-handler-memento-as clearwater-snmp-handler-memento clearwater-snmp-handler-astaire clearwater-snmp-alarm-agent-dbg
+RPM_NAMES := clearwater-snmp-alarm-agent clearwater-snmp-alarm-agent-debuginfo
```

Comment:
I didn't do this originally because it seems like it would couple rpms to deb, but on further thought I'm not sure it does. I think I'd implement this as: 

* `RPM_NAMES` defaults to `PKG_NAMES` but with the `-dbg` -> `-debuginfo` substitution applied. If you ever want to specify `RPM_NAMES` explicitly you have to specify debug packages using `-debuginfo`
* `cw-rpm.mk` spots packages ending in `-debuginfo` and doesn't try to build them. 

Sound OK?